manifests/plugin.pp

```diff
@@ -20,4 +28,12 @@
       order   => '10',
     }
   }
+
+  if $http_content or $https_content {
+    # TODO: prio below main
```

Comment:
I wouldn't know what to do with this TODO. Could you expand on the details?

Reply:
I *think* this is where I need to solve the issue you saw in fpc where the content was added after the main reverse proxy so they ended up not being proxied.